servant-foreign/servant-foreign.cabal

```diff
@@ -1,5 +1,5 @@
 name:                servant-foreign
-version:             0.11.1
+version:             0.11.2
```

Comment:
@alpmestan should we copy https://github.com/phadej/github/blob/master/CONTRIBUTING.md#miscellaneous to our `CONTRIBUTING.md`

In this case next version MUST be `0.12`, as `Req` constructror is exported from non-`Internal` module, so this is a *breaking change*. Additionally, if there happen to be another breaking change, we don't want people "blindly" bump it to `0.13` and so on.

Reply:
Yeah, I guess we want to tell people not to update the version and changelogs.

Reply:
I assumed the Req constructor was not exported.  It shouldn't be...  https://www.yesodweb.com/blog/2011/10/settings-types .  Anyway it's moot as I just reverted all changes to versions and dependencies.